CONTRIBUTING.md

```diff
@@ -16,27 +16,27 @@ You will find tests for files colocated with `*.test.js` suffixes. Whenever maki
 
 If you are adding a new feature or some extra functionality, you should also make sure to accompany those changes with appropriate tests.
 
-### Linting and Formating
+### Linting and Formatting
 
-Before commiting any changes, be sure to do `yarn run lint`; this will lint all relevant files using [ESLint](http://eslint.org/) and report on any changes that you need to make.
+Before committing any changes, be sure to do `yarn run lint`; this will lint all relevant files using [ESLint](http://eslint.org/) and report on any changes that you need to make.
 
-You will also want to ensure your code meets the prettier formating guidelines by running `yarn run prettier -l <filename>` on a specific file. If there are differences the script errors out. You can also speciify a glob `yarn run prettier -l "src/**/*.js"` which will return a list of files that do not conform.
+You will also want to ensure your code meets the prettier formatting guidelines by running `yarn run prettier -l <filename>` on a specific file. If there are differences the script errors out. You can also specify a glob `yarn run prettier -l "src/**/*.js"` which will return a list of files that do not conform.
 
-Alternitivley install the Prettier [editor plugin](https://prettier.io/docs/en/editors.html) in your favorite editor. This is the perfered method.
+Alternatively, install the Prettier [editor plugin](https://prettier.io/docs/en/editors.html) in your favorite editor. This is the preferred method.
 
-There is also a pre-commit hook in place to lint all staged files. If any of the staged files do not conform to the eslint rules or the [prettier](https://prettier.io/) formating guidelines, your commit will fail until you resolve all outstanding issues.
+There is also a pre-commit hook in place to lint all staged files. If any of the staged files do not conform to the eslint rules or the [prettier](https://prettier.io/) formatting guidelines, your commit will fail until you resolve all outstanding issues.
 
-To resolve/fix prettier formating problems from the CLI:
+To resolve/fix prettier formatting problems from the CLI:
```

Comment:
so we actually have a `yarn prettier-fix` which will do the trick more easily than adjusting the individual files, we may want to update this section to basically have them run `lint` and `prettier-fix`

Reply:
I also for later still want to consider adding `prettier-fix` to the pre-commit hook but we've already chatted about that 🙂

Reply:
@ebrillhart -- I tend to vote against any pre-commit hooks, as a personal preference. The wait for any commit to succeed adds up as number of files increases and number of things fixes increases. (Maybe because I commit a lot and squash later). Fine to do whatever here, but I'd rather manually fix and have CI / a runnable check I run before final PR-ing fail me.

Reply:
@ryan-roemer An issue we run into a lot is people PR, CI fails, and then the submitters are unresponsive to requests to fix lint or tests or whatever and repush, which usually means duplicating the work to actually make the fix, since we don't have the same access to forked branches. I'd love to try to prevent that particular scenario, but am certainly open to alternatives to the pre-commit hook!

Reply:
I guess just having in the checklist that people should run the relevant scripts is probably a good compromise and we can revisit if it continues to be an issue!